main.js

```diff
@@ -31,53 +34,25 @@ var DEFAULTS = {
     bindip: '0.0.0.0'
 };
 var NAME = 'electric-moray';
-var LOG_SERIALIZERS = {
-    err: bunyan.stdSerializers.err
-};
-// We'll replace this with the syslog later, if applicable
 var LOG = bunyan.createLogger({
     name: NAME,
-    level: (process.env.LOG_LEVEL || 'info'),
-    src: true,
+    level: 'info',
     stream: process.stderr,
-    serializers: LOG_SERIALIZERS
-});
-var LOG_LEVEL_OVERRIDE = false;
-
-
-function setupLogger(config) {
-    var cfg_b = config.bunyan;
-    assert.object(cfg_b, 'config.bunyan');
-    assert.optionalString(cfg_b.level, 'config.bunyan.level');
-    assert.optionalObject(cfg_b.syslog, 'config.bunyan.syslog');
-
-    var level = LOG.level();
-
-    if (cfg_b.syslog && !LOG_LEVEL_OVERRIDE) {
-        assert.string(cfg_b.syslog.facility, 'config.bunyan.syslog.facility');
-        assert.string(cfg_b.syslog.type, 'config.bunyan.syslog.type');
-
-        var facility = bsyslog.facility[cfg_b.syslog.facility];
-        LOG = bunyan.createLogger({
-            name: NAME,
-            serializers: LOG_SERIALIZERS,
-            streams: [ {
-                level: level,
-                type: 'raw',
-                stream: bsyslog.createBunyanStream({
-                    name: NAME,
-                    facility: facility,
-                    host: cfg_b.syslog.host,
-                    port: cfg_b.syslog.port,
-                    type: cfg_b.syslog.type
-                })
-            } ]
-        });
+    serializers: {
+        err: bunyan.stdSerializers.err
     }
+});
+
+
+function setupLogger(config, options) {
+    assert.object(config.bunyan, 'config.bunyan');
+    assert.optionalString(config.bunyan.level, 'config.bunyan.level');
+    assert.object(options, 'options');
 
-    if (cfg_b.level && !LOG_LEVEL_OVERRIDE) {
-        if (bunyan.resolveLevel(cfg_b.level))
-            LOG.level(cfg_b.level);
+    if (options.verbose) {
+        LOG = LOG.child({level: 'trace', src: true});
+    } else if (config.bunyan.level) {
+        LOG.level(config.bunyan.level);
```

Comment:
Before this change one could set the log level via all of: config file, `-v` or `-vv`, `LOG_LEVEL` envvar. I simplified this to just: config file or `-v`.  There had been no documented usage of either `-v` or `LOG_LEVEL`.  I also added the Logging section to the readme to cover this and the new log rotation.